src/DevBetterWeb.Web/Controllers/BillingController.cs

```diff
@@ -65,7 +75,22 @@ public ActionResult<Subscription> CreateSubscription(SubscriptionCreateRequest r
       }
       catch (StripeException e)
       {
-        return BadRequest(e);
+        var error = new SubscriptionError(e.Message);
+
+        return error;
+      }
```

Comment:
Since both of these catch blocks do the same thing, is there any reason the try block can't be made bigger, so there aren't duplicate catch blocks?